views/default/input/url.js

```diff
@@ -0,0 +1,20 @@
+define(['jquery'], function ($) {
+	$(document).on('blur', 'input[type="url"].elgg-input-url', function() {
+		var val = $(this).val();
+		if (val === '' || val === undefined) {
+			return;
+		}
+		
+		if ($(this).is('[pattern]')) {
+			return;
+		}
+		
+		var patt = new RegExp(/[a-z]+:\S+/gi);
+		var res = patt.test(val);
+		if (res) {
```

Comment:
`var pattern` please and `res` isn't needed can be done directly in the `if` statement

Reply:
changed it